fillers/eips/eip3860.py

```diff
@@ -525,14 +543,23 @@ def generate_create_opcode_initcode_test_cases(
         expected_gas_usage += PUSH_DUP_OPCODE_GAS
 
     if len(initcode.assemble()) > MAX_INITCODE_SIZE and eip_3860_active:
+        # Call returns 0 as out of gas s[0]==1
+        post[to_address(0x200)] = Account(
+            nonce=1,
+            storage={
+                0: 1,
```

Comment:
```suggestion
                0: 1,
                1: 0,
```
Also check for `s[1]`